### PR TITLE
Added KeyCDN to the CDN list

### PR DESCRIPTION
**Pull request by svenba**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -396,6 +396,16 @@ <h3 id="cdn-paas"><a href="#cdn-paas">CDN &amp; PaaS performance</a></h3>
             <td class="alert">no</td>
             <td class="ok">spdy/3.1</td>
           </tr>
+          <tr>
+            <td>KeyCDN</td>
+            <td class="ok">yes</td>
+            <td class="ok">yes</td>
+            <td class="ok">yes</td>
+            <td class="warn">configurable (static)</td>
```
Comment (igrigorik) [inline index.html]: What's the default?

**Comment by igrigorik** [comment]
Thanks for this! Any chance you guys have more links for some of these features? Docs, etc?

File: index.html (hunk 1 above)
Comment (svenba) [inline index.html]: The default is 8k. Should we add it there as well? Space is bit limited.

**Comment by svenba** [comment]
Thanks for the feedback. Just added a link for OCSP stapling.

**Pull request merged**